### DISCUSSION TITLE
Exclude products from discounts

### DISCUSSION THREAD

**Discussion by jordikroon** [craftcms/commerce#3376] [Ideas]
When creating a discount code it is possible to apply certain conditions to when this discount can be used.
It is possible to include products or categories. But it is not possible to exclude products. It would be a nice feature for us to be added.

For example, we would like to include product category X, Y and Z, except Product A, Product B and Product C.

**Comment by segidev** [comment]
That would be really helpful. Currently we need to select hundreds of Productvariants while we only want to exclude one specific one. It would be great to be able to "blacklist" items from a discount

**Comment by wmdhosting** [comment]
+1

**Comment by lukeholder** [comment]
Thanks for the feedback, we plan to improve discount purchasable selection tool for disounts in the future. Welcome the feedback from others.
